setup.cfg

```diff
@@ -24,7 +24,8 @@ setup_requires =
     setuptools >=40.8
 install_requires =
     nibabel >=2.2.1
-    nipype >=1.2.0
+    niflow-nipype1-workflows == 0.0.1
+    nipype >=1.3.0rc1
```

Comment:
This is not a required change. Might go with:

```suggestion
    nipype >=1.2.0,!=1.2.3
```

Unless you explicitly need something from the RC?

Reply:
Sure, but etelemetry was only added in 1.2.3.

Reply:
Yup - https://circleci.com/gh/poldracklab/sdcflows/271